Documentation/ApiOverview/GlobalValues/Typo3ConfVars/MAIL.rst

```diff
@@ -180,6 +182,51 @@ $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_server']
    *only with transport=smtp* server port of mail server to connect to. port
    defaults to "25".
 
+.. index::
+   TYPO3_CONF_VARS MAIL; transport_smtp_domain
+.. _typo3ConfVars_mail_transport_smtp_domain:
+
+$GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_domain']
+============================================================
+
+.. confval:: transport_smtp_domain
+
+   :Path: $GLOBALS['TYPO3_CONF_VARS']['MAIL']
+   :type: text
+   :Default: ''
+
+   Some smtp-relay-server require to set the domain under which the sender is
+   sending an email. As default the EsmtpTransport from Symfony will use the
+   current domain/IP of the host or container. This will be sufficient for
+   the most servers but some servers require a valid domain is passed. If
+   this isn't done, sending emails via such servers will fail.
```

Comment:
```suggestion
   Some smtp-relay-servers require the domain to be set from which the sender is
   sending an email. By default, the EsmtpTransport from Symfony will use the
   current domain/IP of the host or container. This will be sufficient for
   most servers, but some servers require that a valid domain is passed. If
   this isn't done, sending emails via such servers will fail.
```